iothub/device/src/net451/Common/LegacyClientWebSocketTransport.cs

```diff
@@ -245,50 +220,40 @@ protected override bool CloseInternal()
             try
             {
                 if (Logging.IsEnabled)
-                {
                     Logging.Enter(this, $"{nameof(LegacyClientWebSocketTransport)}.{nameof(CloseInternal)}");
-                }
 
-                var webSocketState = this.webSocket.State;
-                if (webSocketState != IotHubClientWebSocket.WebSocketState.Closed && webSocketState != IotHubClientWebSocket.WebSocketState.Aborted)
+                IotHubClientWebSocket.WebSocketState webSocketState = _webSocket.State;
+                if (webSocketState != IotHubClientWebSocket.WebSocketState.Closed
+                    && webSocketState != IotHubClientWebSocket.WebSocketState.Aborted)
                 {
-                    this.CloseInternalAsync();
+                    _ = CloseInternalAsync();
```

Comment:
Should this be awaited? For now, I'm explicitly indicating we don't care about waiting for it.